### PR TITLE
Add test to duplicate profiles

### PR DESCRIPTION
**Pull request by Ghostkeeper**
This goes into the preferences, finds the "Fine" profile, then duplicates it, giving it a custom name. Then it sees if the duplicated profile is in the list of profiles.

### CODE REVIEW

File: suite_Cura/shared/scripts/PageObjects/PreferencesPage.py [hunk 1]
```diff
@@ -71,8 +75,13 @@ def createProfile(self, profile_name):
...
+        self.click(names.btn_duplicate_profile_confirm)
         
     def getProfileFromList(self, profile):
-        printer_obj = self.replaceObjectProperty(names.pps_profile_item, profile)
-        waitForObject(printer_obj, 5000)
-        return printer_obj
\ No newline at end of file
+        profile_obj = self.replaceObjectProperty(names.pfs_profile_item, profile)
+        waitForObject(profile_obj, 5000)
+        return profile_obj
```
Comment (SatishBiharie93) [inline suite_Cura/shared/scripts/PageObjects/PreferencesPage.py]: Return the reference of the object (which is what waitForObject returns) instead of returning the real name of the object (profile_obj) so like this:return waitForObject(profile_obj, 5000)

File: suite_Cura/shared/steps/PreferencesSteps.py [hunk 2]
```diff
@@ -50,7 +50,15 @@ def step(context, action):
...
+@Step("I select the '|word|' custom profile")
+def step(context, profile_name):
+    preferences.selectProfile(profile_name)
+
 @Then("the profile overview contains '|word|' profile")
-def step(context, expected_pofile):
-    actual_printer = preferences.getPrinterFromList(expected_profile)
+def step(context, expected_profile):
+    actual_profile = preferences.getProfileFromList(expected_profile)
     test.compare(expected_profile, actual_profile['text'])
```
Comment (SatishBiharie93) [inline suite_Cura/shared/steps/PreferencesSteps.py]: After changing getProfileFromList to return a reference of the object, you can assert the profiles by using actual_profile.text

File: suite_Cura/shared/scripts/PageObjects/PreferencesPage.py (hunk 1 above)
Comment (Ghostkeeper) [reply suite_Cura/shared/scripts/PageObjects/PreferencesPage.py]: Fixed: https://github.com/Ultimaker/Cura-squish-tests/pull/4/commits/f7d3c6e7c40d305ee9587f2dc843d2ea235d4430

File: suite_Cura/tst_Profile_Management/test.feature [hunk 3]
```diff
@@ -8,8 +8,8 @@ Feature: Profile Management
     And I navigate to Profiles in preferences
     And I select Create profile
     And I give the new profile 'custom' name
-    Then the profile overview contains the profile: 'custom'
-
+    Then the profile overview contains the profile: 'custom #2'
+	And I close the preferences
```
Comment (Ghostkeeper) [inline suite_Cura/tst_Profile_Management/test.feature]: Mixing spaces and tabs here.
